src/editor/components/modals/revision-diff.jsx

```diff
@@ -69,8 +114,15 @@ export function mapStateToProps(state) {
   return {
     currentRevisionContent,
     revisionContentFetchStatus,
-    previousRevisionContent
+    previousRevisionContent,
+    canRevert:
+      selectedRevisionId !== undefined &&
+      state.iomd !== currentRevisionContent &&
```

Comment:
technically we might also want to check the title for differences here, but in practice this dialog obscures the title so we're probably best off leaving this as-is for now.